app/controllers/admin/custom_emojis_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Admin
+  class CustomEmojisController < BaseController
+    def index
+      @custom_emojis = CustomEmoji.all
```

Comment:
Do you not paginate?

Reply:
It makes no sense to me to have more emojis than 40 or 50 at most, so no pagination required

That being said I realized this should filter by local ones only (for now)

Reply:
okay! and

> local ones only

Certainly.